pkgs/top-level/all-packages.nix

```diff
@@ -11886,7 +11886,7 @@ with pkgs;
 
   squashfsTools = callPackage ../tools/filesystems/squashfs { };
 
-  squashfs-tools-ng = callPackage ../tools/filesystems/squashfs-tools-ng { };
+  squashfs-tools-ng = darwin.apple_sdk_11_0.callPackage ../tools/filesystems/squashfs-tools-ng { };
```

Comment:
I find it kinda weird that a callpackage from darwin is used even though it's for all platforms. Why does it look like this?

Reply:
This is actually documented [here](https://github.com/NixOS/nixpkgs/blob/25ccd71ff83a1b6e961588ee65c9f18a7c8f0a92/doc/stdenv/platform-notes.chapter.md).

Reply:
Thanks for referencing that! It seems like a strange design choice, but I guess you were not involves in that.